less/home-page.less

```diff
@@ -70,6 +107,17 @@
         height: auto;
         border-radius: 10px;
       }
+
+      @media @phone {
+        width: 100%;
+        margin: 20px 0;
+      }
+    }
+
+    @media @phone {
+      flex-direction: column-reverse;
+      margin-top: 20px;
+      padding: 15px;
```

Comment:
Also hard to tell what element this media query is for.  you might want to do the indenting for your less so it's easier to read.  Or use a VScode extension like Prettier to do it for you.
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode